service-api/module/Application/src/Model/Service/AccountCleanup/Service.php

```diff
@@ -239,24 +243,27 @@ private function deleteUnactivatedAccounts()
 
     /**
      * @param array $config
+     * @psalm-api
```

Comment:
I got caught up trying to figure out why this isn't called for a while. I think it's because `ServiceAbstractFactory` uses magic, and the ServiceBuilders aren't actually used. We should try to tidy that up with some proper factories at some point. I'll write up a ticket for that, but wanted to run it by you in case you'd noticed anything else.